docs/courses/level-one/README.md

```diff
@@ -0,0 +1,56 @@
+# Introduction
+
+Welcome to the n8n Certification Course Level I!
+
+## Is this course right for me?
+
+This course teaches you how to automate processes with no code using n8n and is meant to jumpstart your qualification as an automation expert.
+
+This course is for you if you:
+- Want to automate things in your personal life
+- Are looking for quick ways to automate common business processes
+- Plan to become an n8n partner
+
+## What will I learn?
+
+By the end of this course, you will know:
+
+- How to set up n8n and navigate the Editor UI
+- How data is structured in n8n
+- How to configure different node parameters and add credentials
+- When and how to use conditional logic in workflows
+- How to schedule and control workflows
+- How to import, download, and share workflows with others
+
+You will build two workflows:
+
+- A two-node workflow to get articles from Hacker News
+- A seven-node workflow to help your client get records from a data warehouse, filter them, make calculations, and notify team members about the results
+
+## How will I learn?
+
+We believe in learning by doing. You can expect some theoretical information about the basic concepts and components of n8n, followed by practice of building workflows step by step.
+
+Throughout the course, we will use some symbols to highlight information:<br/>
+
+⚠️ Warning, exception<br/>
+💡 Tip, shortcut<br/>
+📖 Definition, further reading<br/>
+
+## What do I need to get started?
+
+- A computer with macOS, Windows, or Linux
+- Some basic knowledge of JavaScript and [APIs](https://n8n.io/blog/what-are-apis-how-to-use-them-with-no-code/) are nice-to-haves
+- Eagerness to learn about the power of low-code automation
+
+## How long does it take to complete the course?
+
+Completing the course should take around two hours. You do not have to complete it in one go, feel free to take breaks and resume whenever you are ready.
+
+## Do I get a certificate?
+
+If you want to earn a certificate to show off your skills to the world, you need to take and pass a quiz at the end of the course.
```

Comment:
If you want to earn a certificate to show off your skills to the world, you need to sign up [here](typeform-link). At the end of the course, take and pass a quiz.